frontend/lib/pages/home.dart

```diff
@@ -0,0 +1,267 @@
+// ignore_for_file: prefer_const_constructors
```

Comment:
Only comments I have is potentially make the event you hover over do something like to emphasize that the user is hovering over it.

Reply:
Stylistic choice can be talked about Felix, have the events centered? Whatever you guys think looks best.